build-elements-eik.js

```diff
@@ -0,0 +1,59 @@
+import esbuild from 'esbuild'
+import * as eik from '@eik/esbuild-plugin'
+import pkg from 'glob'
+const { glob } = pkg
+
+await eik.load()
+const imports = glob.sync('packages/**/index.js')
+
+imports.forEach(async (item) => {
+  const regex = /\/(\w+)\//
+  const match = item.match(regex)
+
+  if (item.includes('utils')) return
```

Comment:
Not sure it's relevant, but should we also export `unstyled-heading` and `expand-transition` that's part of `packages/utils/`? In the [./index.js](https://github.com/warp-ds/elements/blob/next/index.js#L16) file we do export them.

Reply:
No one uses these outside the elements repo, hence we are not building them. I think utils are meant for internal use in the repo. But correct me if I'm assuming wrong here :)

Reply:
Makes sense to treat them as internal components unless someone requests access to them, but at the same time our NPM package and the complete package you publish to Eik includes them. I also found one example of expand-transition usage from Fabric (https://finn-codesearch.svc.prod.finn.no/search?q=expand-transition&num=50). I don't mind either approach. I only think it would be good to align on what we make available in Eik and NPM, if that makes sense 😊 

On a side note, it would help review a PR if decisions like this were mentioned in the PR description. It is apparent when one looks deeper into the repo, but with so much going on it can easily be missed 😊